licenses.d2iq.yaml

```diff
@@ -46,22 +46,22 @@ resources:
       - license_path: LICENSE
         ref: v${image_tag}
         url: https://github.com/grafana/loki
-  - container_image: docker.io/istio/install-cni:1.22.3
```

Comment:
do we not install gateway? <pre>
➜  nkp helm repo add test-istio https://istio-release.storage.googleapis.com/charts --kubeconfig=./john-catalog.conf
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: ./john-catalog.conf
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: ./john-catalog.conf
"test-istio" has been added to your repositories
➜  nkp helm --kubeconfig=./john-catalog.conf search repo test-istio
WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: ./john-catalog.conf
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: ./john-catalog.conf
NAME                    	CHART VERSION	APP VERSION	DESCRIPTION                                       
test-istio/ambient      	1.23.2       	1.23.2     	Helm umbrella chart for ambient                   
test-istio/base         	1.23.2       	1.23.2     	Helm chart for deploying Istio cluster resource...
test-istio/cni          	1.23.2       	1.23.2     	Helm chart for istio-cni components               
test-istio/gateway      	1.23.2       	1.23.2     	Helm chart for deploying Istio gateways           
test-istio/istiod       	1.23.2       	1.23.2     	Helm chart for istio control plane                
test-istio/istiod-remote	1.23.2       	1.23.2     	Helm chart for a remote cluster using an extern...
test-istio/ztunnel      	1.23.2       	1.23.2     	Helm chart for istio ztunnel components      
</pre>